schemas/bedrock-profile-http.js

```diff
@@ -159,6 +159,10 @@ const zcap = {
     referenceId: {
       title: 'Reference Id',
       type: 'string'
+    },
+    expires: {
+      title: 'Expiry Date',
+      type: 'string'
```

Comment:
just so you know there are DateTime validators for Expiry Date. If you are short on time i would make an issue to add them at a later date.

https://github.com/digitalbazaar/bedrock-validation/blob/master/schemas/w3cDateTime.js

This is assuming we are using w3c Date time format here.

Reply:
I have updated the schema, good to know about this. Thanks.